packages/fxa-payments-server/src/components/NewUserEmailForm/index.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { NewUserEmailForm } from './index';
+
+describe('NewUserEmailForm test with l10n', () => {
+  it('renders as expected', () => {
+    const { queryByTestId } = render(<NewUserEmailForm />);
+    const form = queryByTestId('new-user-email-form');
+    expect(form).toBeInTheDocument();
```

Comment:
I think we should verify more than just the form element here.  We expect the form to have two email inputs, so we could check for that at least?